README.md

```diff
@@ -1,8 +1,30 @@
 # SPoG-test
-Automation tests for the SPoG UI
 
-This repo deals with UI automation tests for the UI part of the Trustification product.
-The infrastructure is Playwright and the language used is Python.
-Along the way there will be attempts to send these tests to the team running the Trustification repo (https://github.com/trustification/trustification)
-The purpose of this is to allow the Trustification team that is implementing REST API using Rust 
-code, to run the UI tests with their own tests in their pipeline and create a kind of integration tests.
\ No newline at end of file
+SPoG-test is a [Playwright-python](https://playwright.dev/python/ "Playwright-python") based framework to test [trustification](https://github.com/trustification/trustification.git "trustification") UI. 
+
+## Quick Start
+1. Clone the repository 
+
+```
+git clone https://github.com/trustification/spog-test.git
+cd spog-test
+```
+
+2. The shell script [run_spog_tests.sh](run_spog_tests.sh) is capable of setting up the virtual environment and installing the required packages to execute the Automation scripts. At the end of the execution the HTML reports are stored into `results` directory. To run the script, the user should pass trustification application URL, registered Username and Password.
```

Comment:
Maybe we should emphasize that the username and password are 'Trustification' username and password? WDYT?

Reply:
Hmm, it is mentioned that the user should pass trustification application URL, registered Username and Password. Please let me know if this needs to be specified as 'Trustification' username and 'Trustification' password.